test/8_proxy.js

```diff
@@ -0,0 +1,208 @@
+    const prevOracleAddr = await core.oracleAddress();
+    const prevStakingAddr = await core.stakingAddress();
+    const prevConverterAddr = await core.converterAddress();
+    const prevStakingOracleAddr = await tier1Staking.oracleAddress();
+
+    // Get addresses of original logic contracts.
+    const prevWrapperTarget = await wrapper.target();
+    const prevWrapperSushiTarget = await wrapperSushi.target();
+    const prevTokenRewardsTarget = await tokenRewards.target();
+    const prevPlexusOracleTarget = await plexusOracle.target();
+    const prevTier1StakingTarget = await tier1Staking.target();
+    const prevCoreTarget = await core.target();
+    const prevTier2FarmTarget = await tier2Farm.target();
+    const prevTier2AaveTarget = await tier2Aave.target();
+    const prevTier2PickleTarget = await tier2Pickle.target();
+
+    // Re-deploy
+    await updateContracts(wrapper.address, wrapperSushi.address, tokenRewards.address, plexusOracle.address, tier1Staking.address, core.address, tier2Farm.address, tier2Aave.address, tier2Pickle.address);
+
+    describe('', () => {
+      it('The Owner of proxy should remains after updated logic contracts', async () => {
+        expect(await wrapper.owner()).to.equal(prevWrapperProxyOwner);
+        expect(await wrapperSushi.owner()).to.equal(prevWrapperSushiProxyOwner);
+        expect(await tokenRewards.owner()).to.equal(prevTokenRewardsProxyOwner);
+        expect(await plexusOracle.owner()).to.equal(prevPlexusOracleProxyOwner);
+        expect(await tier1Staking.owner()).to.equal(prevTier1StakingProxyOwner);
+        expect(await core.owner()).to.equal(prevCoreProxyOwner);
+        expect(await tier2Farm.owner()).to.equal(prevTier2FarmProxyOwner);
+        expect(await tier2Aave.owner()).to.equal(prevTier2AaveProxyOwner);
+        expect(await tier2Pickle.owner()).to.equal(prevTier2PickleProxyOwner);
+      })
+    })
+
+    //Get address of updated logic contracts.
+    const newWrapperContract = await ethers.getContractAt("WrapAndUnWrap", await wrapper.target());
+    const newWrapperSushiContract = await ethers.getContractAt("WrapAndUnWrapSushi", await wrapperSushi.target());
+    const newTokenRewardsContract = await ethers.getContractAt("TokenRewards", await tokenRewards.target());
+    const newPlexusOracleContract = await ethers.getContractAt("PlexusOracle", await plexusOracle.target());
+    const newTier1StakingContract = await ethers.getContractAt("Tier1FarmController", await tier1Staking.target());
+    const newCoreContract = await ethers.getContractAt("Core", await core.target());
+    const newTier2FarmContract = await ethers.getContractAt("Tier2FarmController", await tier2Farm.target());
+    const newTier2AaveContract = await ethers.getContractAt("Tier2AaveFarmController", await tier2Aave.target());
+    const newTier2PickleContract = await ethers.getContractAt("Tier2PickleFarmController", await tier2Pickle.target());
+
+    describe('', () => {
```

Comment:
Same here remove the empty describe section, this should only be used to literally describe a testing section, so we can't have it blank